symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/parsing/SlashCommandPattern.java

```diff
@@ -149,4 +152,13 @@ private static CommandToken buildCommandToken(String token) {
     }
     return arguments;
   }
+
+  private void checkArgumentNamesUnicity() {
+    List<String> argumentNames = getArgumentNames();
```

Comment:
I would have done this with streams but your code is shorter. This might be useful if we want to show the duplicated arguments in the error message.
```
argumentNames.stream()
        .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()))
        .entrySet()
        .stream()
        .filter(entry -> entry.getValue() > 1)
        .map(Map.Entry::getKey)
        .collect(Collectors.toList());
```